openupgradelib/openupgrade.py

```diff
@@ -2239,13 +2239,17 @@ def format_message(f):
         columns = (
             getattr(model, '_columns', False) or getattr(model, '_fields')
         )
+
+        localdict = {'uid': env.uid}
```

Comment:
```suggestion
        globaldict = {'uid': env.uid}
```
![Selection_245](https://user-images.githubusercontent.com/25005517/156407370-9d3697de-c8a8-46a6-9491-48fff29b3170.png)

Reply:
@MiquelRForgeFlow I updated, thank you